pkg/httputil/httputil.go

```diff
@@ -62,3 +61,13 @@ func IsFiltered(whiteList string, feedState model.FeedState) bool {
 	}
 	return whiteList == string(feedState)
 }
+
+// IsContainNonASCII return true if a string contain non-ascii char
+func IsContainNonASCII(s string) bool {
```

Comment:
Why do we need to put that method in httputil and maybe we need some tests?

Reply:
ok, i will add some tests.
put it into httputil is because it may be use in http api.